prow/metrics/metrics_test.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/test-infra/prow/config"
+)
+
+func TestMetricsPusherIntervals(t *testing.T) {
```

Comment:
This test basically tests that the Go runtime is triggering our `case` with a timer correctly. I'm not sure if this is a useful test.